### DISCUSSION TITLE
Set password for each domain in the account (customers domains)

### DISCUSSION THREAD

**Discussion by wolfganghofer** [plausible/analytics#634] [Anything Else]
Hi! It would be great if I could set a password for every domain in my account so that I, as a webdesign agency, could include my customers domains in my account and give them the possibility to see just their own statistics. 
The email reports are a possibility to give them a summary of their statistics, but that isn´t realy the same thing.
Or maybe you don't even want plausible to be used that way?

**Comment by metmarkosaric** [comment]
thanks @wolfganghofer! you can already share individual dashboards privately with a secure and password protected link. see https://plausible.io/docs/shared-links

**Comment by wolfganghofer** [comment]
Oh sorry, I didn´t see that.
Thanks for your great support!

**Reply by metmarkosaric** [reply]
np! and you're welcome!
